hack/verify-archives.sh

```diff
@@ -16,7 +16,7 @@
 
 # About:
 # This script extracts the golden image tarballs and compares their contents against the
-# already-tracked golden folder (which is used by Bazel to generate deterministic images
+# already-tracked golden folder (which is used by Docker to generate deterministic images
```

Comment:
Docker does not generate deterministic images.

That said, this parenthetical is worth deleting entirely.

Reply:
ack.